fsw/cfe-core/src/evs/cfe_evs_task.c

```diff
@@ -212,8 +212,8 @@ int32 CFE_EVS_CleanUpApp(CFE_ES_ResourceID_t AppID)
 */
 void CFE_EVS_TaskMain(void)
 {
-    int32 Status;    
-    CFE_SB_MsgPtr_t    EVS_MsgPtr; /* Pointer to SB message */
+    int32              Status;
+    CFE_MSG_Message_t *MsgPtr; /* Pointer to SB message */
```

Comment:
Should this comment should be change to not mention SB?
```suggestion
    CFE_MSG_Message_t *MsgPtr; /* Pointer to message */
```

Reply:
Planning to address as part of #1009 cleanup.